package/utils/lua5.3/Makefile

```diff
@@ -108,7 +123,14 @@ define Build/InstallDev
 	$(INSTALL_DIR) $(1)/usr/include/lua5.3 $(1)/usr/lib
 	$(CP) $(PKG_INSTALL_DIR)/usr/include/lua5.3/lua{,lib,conf}.h $(1)/usr/include/lua5.3/
 	$(CP) $(PKG_INSTALL_DIR)/usr/include/lua5.3/lauxlib.h $(1)/usr/include/lua5.3/
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/liblua5.3.a $(1)/usr/lib/
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/liblua5.3.{a,so*} $(1)/usr/lib/
+	$(LN) liblua5.3.so.$(PKG_VERSION) $(1)/usr/lib/liblualib5.3.so
```

Comment:
This results in `liblualib5.3.so` being a symlink to `liblua5.3.so.5.3.5`. The later one doesn't exist.

Reply:
@rmilecki - I've resolved this by linking the actual library version.

Reply:
@rmilecki - One other thing to note, I followed the ubuntu versioning of the so file (0.0.0).